cpp/include/cudf/aggregation.hpp

```diff
@@ -382,6 +384,28 @@ std::unique_ptr<Base> make_argmax_aggregation();
 template <typename Base = aggregation>
 std::unique_ptr<Base> make_argmin_aggregation();
 
+/**
+ * @brief Factory to create a MAX_BY aggregation
+ *
+ * `MAX_BY` returns the value of the element in the group that is the minimum
+ * according to the order_by column.
```

Comment:
What is the `order_by` column here?
```suggestion
 * `MAX_BY` returns values of the first child column that correspond to the minimum values of the second child column.
```

Reply:
updated.